stl/inc/algorithm

```diff
@@ -10359,6 +10367,19 @@ namespace ranges {
             _STL_INTERNAL_STATIC_ASSERT(sentinel_for<_Se2, _It2>);
             _STL_INTERNAL_STATIC_ASSERT(indirect_strict_weak_order<_Pr, projected<_It1, _Pj1>, projected<_It2, _Pj2>>);
 
+            using _Memcmp_classification_pred =
+                typename decltype(_Lex_compare_memcmp_classify(_First1, _First2, _Pred))::_Pred;
+            if constexpr (!same_as<_Memcmp_classification_pred,
+                              void> && sized_sentinel_for<_Se1, _It1> && sized_sentinel_for<_Se2, _It2> //

```

Comment:
Style nitpick: I believe `is_void_v` would be slightly simpler to read. (It additionally accepts cv void which isn't an issue here - (1) we control this type and (2) even if cv void got through, it would cause an error in `_Memcmp_classification_pred{}(_Ans, 0)` below.)